client/services/FeedbackGeneratorService.js

```diff
@@ -54,11 +54,12 @@ tie.factory('FeedbackGeneratorService', [
     };
 
     return {
-      getFeedback: function(prompt, codeEvalResult) {
+      getFeedback: function(
+          prompt, codeEvalResult, rawCodeLineIndexes) {
```

Comment:
Sorry, I mean FeedbackGeneratorServiceSpec.js -- all the calls to getFeedback don't use rawCodeLineIndexes.